src/components/Admin/UserPage/ManageUsers.tsx

```diff
@@ -31,6 +92,45 @@ const ManageUsers: () => JSX.Element = () => {
           textAlign="center"
         >
           <Heading>Manage Users</Heading>
+          <Table>
```

Comment:
A few lines above this (maxWidth="500px"): 

Consider changing this to 800px or something larger than 500px so that on a computer the names don't take up 2 lines (then each row is skinnier, easier to see more data in my opinion.

Reply:
I agree, I changed it to 1000px but let me know if you think a different maxWidth would look better